compare-evidence-strings/README.md

```diff
@@ -60,14 +60,16 @@ First two sections describe statistics per input file. The evidence strings are
 
 For the EVA/ClinVar use case, the association fields are:
 1. ClinVar RCV record accession
-1. Ontology term specifying which phenotype/trait is contained in the record
-1. Allele origin (germline or somatic)
-1. Variant ID (RS ID, if present, or the same RCV ID as in the first field)
+1. Ontology term specifying which phenotype/trait is contained in the record (the ID if mapped, or the original string from ClinVar)
+1. Datatype ID (`genetic_association` or `somatic_mutation`)
+1. Variant ID (`chr_pos_ref_alt`, if present, or the same RCV ID as in the first field)
```

Comment:
I see that most of these "Association fields" match those that we agreed to check uniqueness upon (see [section](https://github.com/EBIvariation/eva-opentargets/blob/fbf59c9f1083478af8a29a31951f6c8f19d665b8/docs/generate-evidence-strings.md#check-that-generated-evidence-strings-do-not-contain-any-duplicates)), but I noticed that the ``variantFunctionalConsequenceId`` is not being considered as an association field, and instead is expected to change between runs of the pipeline. 

Out of curiosity, is there a reason we are considering this field not to be one of the uniqueness tuple in this case?

Reply:
I was mostly just following what Kirill had there before, but I think I agree that we could make this set of fields the same as the ones from the uniqueness tuple (and in fact we could maybe consolidate some of these checks if we decide to make evidence string comparison part of the "official" protocol).  To be honest I don't think it's actually possible to have evidence where everything in that tuple is the same _except_ the functional consequence ID (I think the pipeline explicitly only chooses the most severe consequence), but it doesn't hurt to include it.

It's worth pointing out that just because a field is used to determine uniqueness doesn't mean we don't expect it to change, for example the phenotype representation changes depending on whether it's an unmapped or mapped evidence string.  This is why I added [this madness](https://github.com/apriltuesday/eva-opentargets/blob/issue-201/compare-evidence-strings/compare.sh#L145), to highlight when the tuple is still unique but a value has changed, because in many cases we can still associate strings based on the uniqueness of the other values.

Not sure if any of this is making sense, the takeaway is I'll add functional consequence ID to this list but continue to also check if it changes.

Reply:
Yeah I agree on the fact that it is very improbable to find an evidence string that shares everything _but_ the functional consequence ID. I was asking just because if we are addressing the uniqueness tuple as a set of fields, but later we don't in another process, we may be creating future headaches. 

So either we took it off from the uniqueness tuple or added it here. I saw you already did the latter, so I guess that settles it 👍